### PR TITLE
Beaker

### PR DESCRIPTION
**Pull request by greg-hellings**
A first attempt at syncrhonous-only Beaker provisioning.

### CODE REVIEW

**Comment by herlo** [comment]
Please also rebase against the master branch to obtain the latest requirements.txt

**Comment by herlo** [comment]
bump @greg-hellings. Any progress on this?

**Comment by greg-hellings** [comment]
@herlo Yes, sorry I've been away from dev for the past 2 weeks - work travel for a week, then vacation for a week. I have addressed most of the issues, I'm just looking to finish up the tear-down implementation.

File: ex_schemas/schema_v2.json [hunk 1]
```diff
@@ -50,7 +50,8 @@
                                    {"$ref": "#/definitions/ex_res"},
```
Comment (greg-hellings) [inline ex_schemas/schema_v2.json]: I can't make heads or tails out of schema_v3, sorry.

**Comment by greg-hellings** [comment]
@herlo I've merged master and added tear-down.

**Comment by greg-hellings** [comment]
OK, that should cover the schema_v3 changes.

**Comment by herlo** [comment]
centos-paas-test

**Comment by arilivigni** [comment]
centos-paas-test

**Comment by arilivigni** [comment]
centos-paas-test

**Comment by arilivigni** [comment]
centos-paas-test

**Comment by arilivigni** [comment]
centos-paas-test

**Comment by greg-hellings** [comment]
Is  this going to be merged?

**Comment by herlo** [comment]
@greg-hellings yes, that is the plan. I'm in training this week and haven't had time to test the teardown. I'll be doing that on Monday. Sorry for the delay and noise.

**Comment by herlo** [comment]
After quite a review of the code, I'm fairly concerned with the way that teardown is happening. The concept is that the output should be used to query the server to be able to teardown the systems without modifying the topology. 

@greg-hellings, you mentioned that there is no good documentation for this process, which is true, as we're breaking new ground on provisiong/teardown. However, there is a model that has been used in every other plugin currently written in linch-pin. 

The timeline is tight, but @samvarankashyap is willing to work with you in the next couple of days to try to rework the beaker module to ensure it can be merged and follow the model of the other plugins. Would that be something you would be willing to do right away??

If not, I can continue down the path of reworking the module, but I'm definitely not as familiar with beaker, and it may delay any plans you have with a presentation. Alternatively, you could just use this fork/branch for any demonstrations.

Thoughts?

**Comment by greg-hellings** [comment]
I don't have time allocated in the current sprint to continue chasing a set of hazily moving goal posts with this. If you want drastic changes like that from me, I'll schedule them into a sprint once the procedure and features needed are clearly documented. Otherwise, you can do as you wish with this branch.

**Pull request merged**